components/server/src/prebuilds/prebuild-manager.ts

```diff
@@ -77,21 +77,18 @@ export class PrebuildManager {
             const results: Promise<any>[] = [];
             for (const prebuild of prebuilds) {
                 try {
-                    for (const instance of prebuild.instances) {
```

Comment:
No clue why we would cycle over instances here before. I could imagine this was to mitigate some weird behavior we had in the past. :thinking:
We only every have one WorkspaceInstance for every Workspace of type = 'prebuilt'. And even if we'd change that in the future: There is always ever one WorkspaceInstance "running" for each given Workspace. :shrug: